orchestrator-salt/src/main/resources/salt/pillar/postgresql/disaster_recovery.sls

```diff
@@ -1,3 +1,4 @@
 disaster_recovery:
   object_storage_url:
+  aws_region: '' # empty string to prevent salt from passing the literal 'None'
```

Comment:
When/where is this actually filled in with the region, if applicable?

Reply:
you are defining a new pillar for this, right. Where is the application change that passes this information?

Reply:
y'all are correct. I had originally written the issue to just include the salt script changes, but obviously we need a way to provide the region value.

I'm investigating how to determine the appropriate region and add it to the pillar.

Reply:
I've pushed updated changes to the salt pillar config.